quizgen/latex.py

```diff
@@ -3,32 +3,81 @@
 import shutil
 import subprocess
 
+import quizgen.util.dirent
+
 _pdflatex_bin_path = None
+_pdflatex_use_docker = False
+
+DOCKER_IMAGE = "quizgen/latex.py"
 
 def set_pdflatex_bin_path(path):
     global _pdflatex_bin_path
     _pdflatex_bin_path = path
 
-def is_available():
-    if (_pdflatex_bin_path is not None):
-        return True
-
-    if (shutil.which('pdflatex') is None):
-        logging.warning("Could not find `pdxlatex`, cannot compile PDFs")
-        return False
+def set_pdflatex_use_docker(use_docker):
+    global _pdflatex_use_docker
+    _pdflatex_use_docker = use_docker
 
-    return True
+def is_available(use_docker = False):
```

Comment:
`use_docker` shouldn't be passed in here.
`_pdflatex_use_docker` and `_pdflatex_bin_path` should be used.